.github/workflows/ci.yml

```diff
@@ -15,15 +15,11 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9]
```

Comment:
I'd go one step further and format as
```yaml
        python-version: [
            '3.7',
            '3.8',
            '3.9',
        ]
```
so that git blame can be utilised to the fullest in the future if we need to pin point when changed occurred